From: Angelos Oikonomopoulos <angelos@igalia.com>
Local hack to work around
    cc1: error: '-mfloat-abi=hard': selected architecture lacks an FPU
error.
diff --git a/util/perl/OpenSSL/config.pm b/util/perl/OpenSSL/config.pm
--- a/util/perl/OpenSSL/config.pm
+++ b/util/perl/OpenSSL/config.pm
@@ -667,8 +667,8 @@
       ],
       [ 'armv[1-3].*-.*-linux2',  { target => "linux-generic32" } ],
       [ 'armv[7-9].*-.*-linux2',  { target => "linux-armv4",
-                                    cflags => [ '-march=armv7-a' ],
-                                    cxxflags => [ '-march=armv7-a' ] } ],
+                                    cflags => [ '-march=armv7-a+vfpv4' ],
+                                    cxxflags => [ '-march=armv7-a+vfpv4' ] } ],
       [ 'arm.*-.*-linux2',        { target => "linux-armv4" } ],
       [ 'aarch64-.*-linux2',      { target => "linux-aarch64" } ],
       [ 'sh.*b-.*-linux2',        { target => "linux-generic32",
